vizro-core/pyproject.toml

```diff
@@ -54,7 +54,7 @@ exclude_lines = [
   "if __name__ == .__main__.:",
   "if TYPE_CHECKING:"
 ]
-fail_under = 91
+fail_under = 90
```

Comment:
Just lowering it temporarily. Should go up again as soon as the PR on the test containers are merged + the tests I am writing for the tabs separately.

Reply:
Increased again here: https://github.com/mckinsey/vizro/pull/291/commits/799790a8fae837b9246b675fd48721cabc319cba